### ISSUE TITLE
Conflicting instructions about collections

### ISSUE THREAD

**Issue by asarkar** [mmistakes/minimal-mistakes#4982]
### Motivation

https://mmistakes.github.io/minimal-mistakes/docs/collections/

In `_pages/portfolio.md`
```
title: Portfolio
layout: collection
permalink: /portfolio/
collection: portfolio
entries_layout: grid
classes: wide
```

https://mmistakes.github.io/minimal-mistakes/docs/configuration/#paginate-v2

```
---
title: "Lovely pets"
layout: home
permalink: /pets/
pagination:
  enabled: true
  collection:
    - cat
    - dog
---
```

One uses a `collection` layout, and another `home`. Also, since each collection has its own page under `_pages`, how does the latter list two collections? What is the location and name of the file where the second front matter should go?

### Suggestion

Clarify.

**Comment by iBug** [comment]
99% of time there's something wrong with your config. If you don't provide a [minimal, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) there's nothing we can help.

**Comment by asarkar** [comment]
I understand the importance of a minimal, reproducible example, but in this case, I pointed out the discrepancies in the documentation, and asked specific questions linking to relevant parts of the documentation. I don’t know what sort of code would be useful here, and why any would be necessary.

Here are the questions again, somewhat rephrased:
1. It appears that in order to paginate all the posts, either a `index.md` at the base of the site, or a `_pages/posts-index.md` would work. Either file should specify the collection `posts`.
2. It appears that each custom collection that is not a post can be paginated by a corresponding `_page/custom-collection-index.md`, or a single `_page/collections-index.md` that lists all the collections.
3. Jekyll archive pages like `_pages/tag-archive.md` and `_page/collection-archive.md` could conflict with pagination if the permalinks are the same.

I looked at the `iobugs.io` source which was referenced in one of the tickets as using pagination. The `pagination` section in its `_config.yml` doesn’t exactly match what’s documented in minimal-mistakes, as shown above.

I hope my confusion is clear now.

**Comment by iBug** [comment]
Points here are:

1. The `collection` layout came out much earlier and was intended for indexing a single collection. On the other hand, the Paginate V2 part is to showcase support for Paginate V2 and is not specifically intended for collections.
2. Collections do not have to be indexed, so it's up to the user to create an index page for each collection they want.
3. Permalink conflict is probably out of scope for a theme. It's a Jekyll problem (or more generally, any CMS that allows custom permalinks but fails to enforce uniqueness).
4. `_pages` is just a special collection named `pages`. You're confusing this with [`collections_dir` setting](https://jekyllrb.com/docs/collections/).

**Comment by asarkar** [comment]
I think I figured out how to use the jekyll-paginate-v2, although to be honest I found the minimal-mistakes documentation in this regard lacking. github.com/iBug and jekyll-paginate-v2 examples were helpful to me.

One last question before I close the ticket: I found that the pagination pages using minimal-mistakes don't process markdown, and the content have to be HTML. For example, the following content was rendered as-is.
```
[Test](/test)
```
However, jekyll-paginate-v2 has many examples where the pagination pages are written in markdown. Is this a known limitation?
